CONTRIBUTING.md

```diff
@@ -0,0 +1,167 @@
+# Sending a pull request
+
+1. Fork the repository
```

Comment:
nice addition, for those who never contributed to open source, forking a repo is not obvious :slightly_smiling_face: